### PR TITLE
Unit testing oasisctl

### PR DESCRIPTION
**Pull request by Skarlso**

### CODE REVIEW

File: tests/crypto/crypto_create_test.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+Created-At                 now
+Deleted-At                 -
+$`
+	out, err := tests.RunCommand(args)
+	require.NoError(t, err)
+
+	certId, err := tests.GetResourceID(string(out))
+	require.NoError(t, err)
+	defer func() {
+		conn := cmd.MustDialAPI()
```
Comment (ewoutp) [inline tests/crypto/crypto_create_test.go:63]: Connection must be closed

File: tests/crypto/crypto_delete_test.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+	"github.com/arangodb-managed/oasisctl/cmd"
+	"github.com/arangodb-managed/oasisctl/tests"
+)
+
+func TestDeleteCertificate(t *testing.T) {
+	// Initialize the root command.
+	cmd.RootCmd.PersistentPreRun(nil, nil)
+	ctx := cmd.ContextWithToken()
+	conn := cmd.MustDialAPI()
```
Comment (ewoutp) [inline tests/crypto/crypto_delete_test.go:42]: Close...

**Review by ewoutp — COMMENTED** [review]
Please ensure first that all connections are properly closed

File: tests/platform/list_region_test.go [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+func TestListRegion(t *testing.T) {
+	org, err := tests.GetDefaultOrganization()
+	require.NoError(t, err)
+	args := []string{"list", "regions", "--provider-id=aks", "--organization-id=" + org}
+	compare := `Id                | Provider-Id | Location                 | Available
+aks-canadacentral | aks         | Central Canada, Toronto  | ✓
+aks-eastus2       | aks         | East US, Virginia        | ✓
+aks-uksouth       | aks         | UK, London               | ✓
+aks-westeurope    | aks         | West Europe, Netherlands | ✓
+aks-westus2       | aks         | West US, Washington      | ✓
```
Comment (ewoutp) [inline tests/platform/list_region_test.go:44]: This test will fail as soon as we add1 region. 
Can you make this a bit more flexible

Comment (Skarlso) [reply tests/platform/list_region_test.go:44]: Yes, I can leave room for arbitrary information in between these lines.

**Review by ewoutp — APPROVED** [review]
LGTM

**Pull request merged**